jenkins/ci.suse.de/openstack-ardana-qa-tests.yaml

```diff
@@ -121,6 +121,14 @@
             The Jenkins agent where this job must run. Used by upstream jobs to
             force a job to reuse the same node.
 
+      - choice:
+          name: type_of_compute
+          choices:
+            - RHEL-COMPUTE
+            - SLES-COMPUTE
+          description: >-
+            Type of Compute Node to be removed
+
```

Comment:
Not sure but by default it will pick RHEL. So it is better to leave it empty to make sure it does not impact others.